### ISSUE TITLE
Root directory and /etc should not belong to group warewulf

### ISSUE THREAD

**Issue by HaroldoUFRN** [warewulf/warewulf#1078]
### Steps to reproduce

I'm not sure what exactly did I do to cause it.
Just `ls -lah /` to see it.

### Error message

```
[haroldo@master0 ~]$ ls -lah /
total 24K
dr-xr-xr-x.  18 root warewulf  235 Jan  2 12:43 .
dr-xr-xr-x.  18 root warewulf  235 Jan  2 12:43 ..
lrwxrwxrwx.   1 root root        7 Oct 10  2021 bin -> usr/bin
dr-xr-xr-x.   5 root root     4.0K Dec 13 16:56 boot
drwxr-xr-x.  21 root root     3.2K Feb  6 09:37 dev
drwxr-xr-x.  99 root warewulf 8.0K Feb  6 09:38 etc
```

`/` and `/etc` are now in the warewulf group.

### Information on your system

wwctl version:	 4.4.1-1.git_d6f6fed
rpc version: apiPrefix:"rc1" apiVersion:"1" warewulfVersion:"4.4.1-1.git_d6f6fed
"

NAME="Rocky Linux"
VERSION="8.9 (Green Obsidian)"
ID="rocky"
ID_LIKE="rhel centos fedora"
VERSION_ID="8.9"
PLATFORM_ID="platform:el8"
PRETTY_NAME="Rocky Linux 8.9 (Green Obsidian)"
ANSI_COLOR="0;32"
LOGO="fedora-logo-icon"
CPE_NAME="cpe:/o:rocky:rocky:8:GA"
HOME_URL="https://rockylinux.org/"
BUG_REPORT_URL="https://bugs.rockylinux.org/"
SUPPORT_END="2029-05-31"
ROCKY_SUPPORT_PRODUCT="Rocky-Linux-8"
ROCKY_SUPPORT_PRODUCT_VERSION="8.9"
REDHAT_SUPPORT_PRODUCT="Rocky Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="8.9"

### General information

- [X] I have run `wwctl version` and reported the contents of `/etc/os-release`
- [X] I have searched the issues of this repo and believe this is not a duplicate
- [X] I have captured and reported relevant error messages and logs

**Comment by anderbubble** [comment]
Confirmed in my test environment. Appears to be because the source overlays are owned by the `warewulf` group.

**Comment by mslacken** [comment]
Unfortunately this is the intended and right behavior. Overlays exists for the purpose to set and change the permissions of vital configuration files.
The problem here is that the the files and sub directories have the wrong permissions in the overlay. So this has to changed in the spec and Makfile.

**Comment by mslacken** [comment]
@HaroldoUFRN You have to change the ownership under `/var/lib/warewulf/overlays/host`  to match the ones you want. This will fix the problem.

**Comment by anderbubble** [comment]
I don't think the overlays should have group `warewulf` ownership in this case; they just do because it's the default in the `%files` section, and it's probably my fault for messing it up when I updated the specfile in the past.

I should be able to get these packaged with the correct / less-surprising permissions.

**Comment by griznog** [comment]
Adding the `rootfs` directory to the overlay to act as the eventual `/` allows setting its permissions properly while still allowing the overlay directory to have `warewulf` as the group and retain restrictive permissions at any site that doesn't want overlays visible.

**Comment by JasonYangShadow** [comment]
yeah, after running `wwctl configure --all`, some files/dirs' permission will be updated. (screenshot is just an example)
![image](https://github.com/warewulf/warewulf/assets/2051711/d32732f3-35f6-46be-928b-377cb4a1ec56)
![image](https://github.com/warewulf/warewulf/assets/2051711/47780158-63ba-403a-a4f1-153d5b06f5db)

**Comment by griznog** [comment]
All my overlays use `overlayname/rootfs/` so that the overlay content is in `rootfs`. The `overlayname` directory is `root:warewulf` and `overlayname/rootfs` is `root:root`. The result on a booted node is:

```
[root@cpu-c-1 ~]# ls -al /
total 181
dr-xr-xr-x   23 root root 1500 Feb 10 08:51 .
dr-xr-xr-x   23 root root 1500 Feb 10 08:51 ..
-rw-r--r--    1 root root    0 Nov 15  2021 .autorelabel
lrwxrwxrwx    1 root root    7 Oct 10  2021 bin -> usr/bin
dr-xr-xr-x    5 root root  220 Nov 28 16:17 boot
drwxr-xr-x    6 root root    6 Oct 26  2022 datapool
drwxr-xr-x   21 root root 3480 Jan 20 08:30 dev
drwxr-xr-x  224 root root 8280 Feb 10 08:51 etc
drwxr-xr-x    4 root root    0 Feb 14 03:40 home
drwxr-xr-x   16 root root    0 Jan 15 16:16 hpc
lrwxrwxrwx    1 root root   24 Jan 15 08:18 init -> /usr/lib/systemd/systemd
lrwxrwxrwx    1 root root    7 Oct 10  2021 lib -> usr/lib
lrwxrwxrwx    1 root root    9 Oct 10  2021 lib64 -> usr/lib64
drwxr-xr-x    3 root root   60 Jan 15 16:16 local
drwxr-xr-x    2 root root   40 Oct 10  2021 media
drwxr-xr-x    2 root root    0 Jan 15 16:16 misc
drwxr-xr-x    2 root root   40 Oct 10  2021 mnt
drwxr-xr-x    2 root root    0 Jan 15 16:16 net
drwxr-xr-x   10 root root  200 Jan 15 16:18 opt
dr-xr-xr-x 1732 root root    0 Jan 15 16:15 proc
lrwxrwxrwx    1 root root   19 Jan 15 16:16 programs -> /hpc/sbgrid/current
drwxr-xr-x    9 root root  380 Sep 25 09:39 root
drwxr-xr-x   64 root root 1900 Jan 15 17:08 run
lrwxrwxrwx    1 root root    8 Oct 10  2021 sbin -> usr/sbin
drwxr-xr-x    2 root root   40 Oct 10  2021 srv
dr-xr-xr-x   13 root root    0 Jan 15 16:15 sys
drwxrwxrwt    7 root root  140 Feb 14 06:58 tmp
drwxr-xr-x   15 root root  320 Apr 15  2023 usr
drwxr-xr-x   25 root root  580 Jan 15 16:23 var
-rw-r--r--    1 root root    3 Dec 26 06:24 .version
drwxr-xr-x   11 root root  220 Jan 15 16:15 warewulf
```

The shipped overlays just need to have content moved into a `rootfs` directory in the overlay directory.
